src/plugins/SpatialProfile.h

```diff
@@ -90,6 +90,10 @@ class SpatialProfile : public PluginBase {
 
 	SamplingInformation samplInfo;
 
+	void saveCallbacksLocally(const std::map<std::string, FunctionWrapper>& callbackMap) override {
+		getNumFixRegion = callbackMap.at("FixRegion::getMoleculesInRegion").get<unsigned long>();
```

Comment:
State where this callback comes from, so basically on what plugin this one depends. We probably still need some kind of plugin dependency check? ;)

Reply:
yes :)